frontend/src/app/containers/ExperimentPage/DetailsHeader.js

```diff
@@ -5,7 +5,7 @@ import classnames from "classnames";
 import { Localized } from "fluent-react/compat";
 import LayoutWrapper from "../../components/LayoutWrapper";
 import ExperimentPlatforms from "../../components/ExperimentPlatforms";
-import { experimentL10nId } from "../../lib/utils";
+import { experimentL10nId, isMobile } from "../../lib/utils";
```

Comment:
I imagine the modal issue #3509 will also want to distinguish iOS from Android. Maybe `isIOS` and `isAndroid` should go in utils?

Reply:
I agree, I'll move it when I address #3509